src/main/java/com/cloudbees/plugins/credentials/GlobalCredentialsConfiguration.java

```diff
@@ -68,6 +68,7 @@
     /**
      * Our filter.
      */
+    @Deprecated
```

Comment:
Any particular reason this is being deprecated?

Reply:
That's an old constant using the Guava `Predicate` interface which was deprecated (and marked with a `@DoNotUse` which failed the build otherwise). The update just uses an inline SAM to align to the new method that uses the `java.util.function` version.

Reply:
The deprecated code was the `Functions` method by the way, not Guava itself.

Reply:
…because you need a `Predicate<Descriptor>` now.

Reply:
The field is public. I didn't want to remove it right away, and the field is useless ever since I refactored it to use a lambda instead of an anonymous class anyways.

Reply:
Oh I see what you mean.